<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Jäsenyys</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="editMembershipJoinedDateEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ryhmä</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Jäsen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Liittyi</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="editMembershipExitDateEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="editMembershipShareSpinBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>50</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Osuus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editMembershipCancelPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="editMembershipSavePushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tallenna</string>
   </property>
  </widget>
  <widget class="QComboBox" name="editMembershipGroupComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="editMembershipMemberComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="editMembershipPopulatePushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Täytä</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="editMembershipTableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>631</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="editMembershipExitCheckBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Poistuu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurue</string>
   </property>
  </widget>
  <widget class="QComboBox" name="editMembershipPartyComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editMembershipTableWidget</tabstop>
  <tabstop>editMembershipPopulatePushButton</tabstop>
  <tabstop>editMembershipGroupComboBox</tabstop>
  <tabstop>editMembershipMemberComboBox</tabstop>
  <tabstop>editMembershipJoinedDateEdit</tabstop>
  <tabstop>editMembershipExitCheckBox</tabstop>
  <tabstop>editMembershipExitDateEdit</tabstop>
  <tabstop>editMembershipShareSpinBox</tabstop>
  <tabstop>editMembershipCancelPushButton</tabstop>
  <tabstop>editMembershipSavePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
